collector/src/bin/collector.rs

```diff
@@ -628,6 +630,18 @@ enum Commands {
         #[command(flatten)]
         db: DbOption,
     },
+
+    /// Displays diff between two local bench result.
+    BenchLocalDiff {
+        #[command(flatten)]
+        db: DbOption,
+
+        #[arg(long)]
+        a_id: String,
```

Comment:
We want to have some notion of a base and a modified artifact, so I would call these `base` and `modified`.
Ideally, it the arguments should be long, but also positional, so that it is possible to do `bench_cmp a b` or `bench_cmp --base a -- modified b`.